### DISCUSSION TITLE
Authentication

### DISCUSSION THREAD

**Discussion by gerichkoeke** [akvorado/akvorado#1087] [Q&A]
Good afternoon, I see that on the akvorado screen there is an icon with the description as if a user was logged in. Is there any way to have a login screen for akvorado? Can I integrate it with ldap?

**Comment by vincentbernat** [comment]
This is in the [documentation](https://demo.akvorado.net/docs/configuration#authentication).

**Comment by Hyperion3003** [comment]
I need help with this, i don't know if I misunderstood the documentation but when I leave the field of user from default to empty like the documentation, I still can enter akvorado and I want to restrict to just log in from auth2 proxy . This is what will decide if I keep using this system or don't because a cant have an open system in my organization

**Reply by vincentbernat** [reply]
It's the job of the authenticating proxy to restrict access to unauthenticated users.
